strawberryfields/apps/qchem/dynamics.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Functions used for simulating vibrational quantum dynamics of molecules.
+
+Photonic quantum devices can be programmed with molecular data in order to simulate the dynamics
+of spatially-localized vibrations in molecules :cite:`sparrow2018simulating`. These vibrational
+quantum dynamics can be simulated with a device that is programmed to implement the transformation:
```

Comment:
Maybe these two sentences can be joined:
"Photonic quantum devices can be programmed with molecular data in order to simulate the **quantum** dynamics of spatially-localized vibrations in molecules :cite:`sparrow2018simulating`. To that aim, the quantum device has to be programmed to implement the transformation: